### DISCUSSION TITLE
How PrimeNG support adaptive? is there a guide?

### DISCUSSION THREAD

**Discussion by ecdavidw** [primefaces/community#494] [PrimeNG]
I feel in general the components are a bit bigger. How do I arrange like 20-30 input, checkbox etc, on a input page? And how to make it still display well on mobile screen? 

thanks

**Comment by ecdavidw** [comment]
I found this, maybe it is the answer?
Form Layout
Form layout is a CSS utility optimized for creating forms with ease. FormLayout is not included in PrimeFaces as it is provided by [PrimeFlex](https://primeflex.org/), a shared grid library between PrimeFaces, PrimeNG, PrimeReact and PrimeVue projects.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please explain what you want to do? Issue is not clear.

**Reply by ecdavidw** [reply]
Thanks. 
For example, I have a edit page, have a bunch fields like 30+ input fields. On computer browser I may display all 30 fields on the page without worrying about the space, I may display two columns or more. But on mobile browser, I can only display less then 10 fields, need scroll bar etc. What components I should use to make the page adaptive to archive that?

**Reply by cetincakiroglu** [reply]
Could you please share a stackblitz example so we can identify the issue clearly?
